src/lib/containers/provenance/ProvenanceUtils.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react'
+import {
+  Node,
+  Position,
+  ConnectionLineType,
+  Edge,
+  MarkerType,
+} from 'react-flow-renderer'
+import { ActivityNodeLabel } from './ActivityNodeLabel'
+import { EntityNodeLabel } from './EntityNodeLabel'
+import {
+  ExpandGraphNodeDataProps,
+  ExpandGraphNodeLabel,
+} from './ExpandGraphNodeLabel'
+import { ExternalGraphNodeLabel } from './ExternalGraphNodeLabel'
+import dagre from 'dagre'
+import _ from 'lodash'
```

Comment:
Switch to `import { ... } from 'lodash-es'` to prevent bloating the build

Reply:
Another nice catch

Reply:
Fixed